cmd/subctl/join.go

```diff
@@ -113,7 +113,7 @@ func addJoinFlags(cmd *cobra.Command) {
 		"enable Submariner pod debugging (verbose logging in the deployed pods)")
 	cmd.Flags().BoolVar(&joinFlags.OperatorDebug, "operator-debug", false, "enable operator debugging (verbose logging)")
 	cmd.Flags().BoolVar(&labelGateway, "label-gateway", true, "label gateways if necessary")
-	cmd.Flags().StringVar(&joinFlags.CableDriver, "cable-driver", "", "cable driver implementation")
+	cmd.Flags().StringVar(&joinFlags.CableDriver, "cable-driver", "", "cable driver implementation (default \"libreswan\")")
```

Comment:
```suggestion
	cmd.Flags().StringVar(&joinFlags.CableDriver, "cable-driver", "libreswan", "cable driver implementation")
```

I agree with @skitt here, if you want a default value set it explicitly here (and as a bonus it also gets printed).

Reply:
The initial version of this PR set the default, and I urged caution since that would involve changing the default codepaths — I didn’t think there was much risk but it would have meant ensuring that all our tests were still OK. Adding the default in the help text allows us to provide the same information without changing the default codepaths.

Reply:
I don't see how that makes any sense actually, if we leave it empty then the default is determined elsewhere and could be different than the "documented" one and as such misleading.

If anything, the safer choice is to set the default here regardless of what the default is in the underlying implementation.

As for tests, theres no real danger in one way or the other, since a default "libreswan" is set somewhere, and if we fear of that then we should set it explicitly, but TBH it doesnt matter whats the default in that case as its all expected to still work.